### DISCUSSION TITLE
Dropdown/Select with Search Box - Available

### DISCUSSION THREAD

**Discussion by jhanily** [Vonage/vivid#776] [General]
Is the Dropdown/Select with Search Box available and ready to be consumed? (Unified Portal)

**Comment by yinonov** [comment]
[select is ready](https://vivid.vonage.com/?path=/story/components-select--default).
I briefly posted on search box and its permutations [here](https://vonage.workplace.com/groups/vividupdates/permalink/748077125879255/)
